src/InteractiveWindow/VisualStudio/IVsInteractiveWindow2.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.InteractiveWindow.Shell
+{
+    /// <summary>
+    /// Extends <see cref="IVsInteractiveWindow"/> with additional functionality.
+    /// </summary>
+    public interface IVsInteractiveWindow2 : IVsInteractiveWindow
+    {
+        /// <summary>
+        /// Gets the <see cref="IVsWindowFrame"/> associated with the <see cref="IVsInteractiveWindow2"/>.
+        /// </summary>
+        object WindowFrame { get; }
```

Comment:
Because the underlying implementation doesn't.  I don't mind casting it, but I assumed there must be a reason.